line-bot-model/src/main/java/com/linecorp/bot/model/event/beacon/BeaconContentUtil.java

```diff
@@ -16,29 +16,51 @@
 
 package com.linecorp.bot.model.event.beacon;
 
-import javax.xml.bind.DatatypeConverter;
-
 final class BeaconContentUtil {
     private BeaconContentUtil() {
         throw new UnsupportedOperationException("This is a utility class and cannot be instantiated");
     }
 
+    private static final char[] HEX_CODE = "0123456789abcdef".toCharArray();
```

Comment:
I suggest to write char array directly. Because it makes class loading time. It's really tiny optimization, but we should do it since this class is a part of model module.